frontend/components/Filter/OrganiserFilterDialogMobile.tsx

```diff
@@ -0,0 +1,339 @@
+"use client";
+
+import { EventData } from "@/interfaces/EventTypes";
+import { AdjustmentsHorizontalIcon, ChevronDownIcon, ChevronUpIcon, XMarkIcon } from "@heroicons/react/24/outline";
+
+import { Fragment, useMemo, useState } from "react";
+import Datepicker from "react-tailwindcss-datepicker";
+import ListBox from "../ListBox";
+import { Transition, Dialog } from "@headlessui/react";
+import { Slider } from "@material-tailwind/react";
+import { InvertedHighlightButton } from "../elements/HighlightButton";
+import { PROXIMITY_SLIDER_MAX_VALUE } from "./FilterDialog";
+
+export enum SortByCategory {
+  HOT,
+  TOP_RATED,
+  PRICE_ASCENDING,
+  PRICE_DESCENDING,
+  DATE_ASCENDING,
+  DATE_DESCENDING,
+}
+
+export const DEFAULT_SORT_BY_CATEGORY = SortByCategory.HOT;
```

Comment:
These constant values are only used in this file and do not need to be exported.
I think because we want one source of truth for these constants, and `OrganiserFilterDialog` is the 'main' version of this functionality, we should import these constants from `OrganiserFilterDialog.tsx` and remove these duplicated constants from this file.